src/RemoteAPI/content-suggestions/class-content-suggestions-base-api.php

```diff
@@ -142,4 +162,31 @@ protected function post_request( array $query = array(), array $body = array() )
 
 		return $decoded;
 	}
+
+	/**
+	 * Truncate the content of an array to a maximum length.
+	 *
+	 * @param string|array|mixed $content The array to truncate.
```

Comment:
```suggestion
	 * @param string|array|mixed $content The content to truncate.
```
Should we change `array` another word, since it can be a string? Same for the line below.